oasis/beagle-strings

```diff
@@ -0,0 +1,12 @@
+Flag beagle_strings
+  Description: Build string analysis library
```

Comment:
It is not a library but a plugin. The description would be `Build the strings plugin`